src/Orc.Snapshots/Extensions/ZipEntryExtensions.cs

```diff
@@ -8,18 +8,33 @@
 namespace Orc.Snapshots
 {
     using System.IO;
-    using Ionic.Zip;
+    using System.IO.Compression;
+    using System.Threading.Tasks;
 
     internal static class ZipEntryExtensions
     {
-        public static byte[] GetBytes(this ZipEntry entry)
+        public static async Task<byte[]> GetBytesAsync(this ZipArchiveEntry entry)
         {
-            using (var memoryStream = new MemoryStream())
+            using (var stream = entry.Open())
             {
-                entry.Extract(memoryStream);
+                using (var memoryStream = new MemoryStream())
+                {
+                    await memoryStream.FlushAsync();
```

Comment:
first copy to, then flush. Flush basically means: "make sure you've written the data"